apps/settings/index.html

```diff
@@ -17,21 +17,77 @@
     <link rel="stylesheet" href="style/index.css" type="text/css" />
 
     <script src="js/config.js"></script>
-    <script src="components/panel_link.js"></script>
+    <script src="js/dependencies.js"></script>
     <script src="js/bootstrap.js"></script>
   </head>
   <body>
-    <div id="main">
-      <input id="search" data-l10n-id="search-settings" />
-      <ul>
-        <panel-link name="wifi" icon="wifi"></panel-link>
-        <panel-link name="telephony" icon="phone"></panel-link>
-        <panel-link name="display" icon="image" disabled></panel-link>
-        <panel-link name="lockscreen" icon="lock"></panel-link>
-        <panel-link name="region_language" icon="globe" disabled></panel-link>
-        <panel-link name="date_time" icon="clock" disabled></panel-link>
-        <panel-link name="system_info" icon="info"></panel-link>
-      </ul>
-    </div>
+    <section id="spinner" style="display: flex; align-items: center; justify-content: center;">
+      <sl-spinner style="font-size: 4rem;"></sl-spinner>
+    </section>
+    <section id="main" class="hidden">
+      <header>
+        <sl-input id="search" clearable data-l10n-id="search-settings" data-l10n-attrs="placeholder">
+          <sl-icon name="search" slot="prefix" />
+        </sl-input>
+      </header>
+      <sl-menu id="main-menu">
+        <sl-menu-label>Connectivity</sl-menu-label>
+        <sl-menu-item value="wifi">
+            <sl-icon name="wifi" slot="prefix"></sl-icon>
+            <sl-icon name="chevron-right" slot="suffix"></sl-icon>
```

Comment:
Not much we can do now, but it looks like Shoelace will provide that as a slot in a future release: https://github.com/shoelace-style/shoelace/blob/8ae987ea691fed9faa1906b88964cc5932809cba/src/components/menu-item/menu-item.ts#L79

Reply:
Until then, we'll do it like this, I guess.